reader/src/main/java/org/jline/reader/impl/LineReaderImpl.java

```diff
@@ -6263,6 +6270,14 @@ public KeyMap<Binding> safe() {
         return safe;
     }
 
+    public KeyMap<Binding> dumb() {
+        KeyMap<Binding> dumb = new KeyMap<>();
+        bind(dumb, SELF_INSERT, range("^@-^?"));
+        bind(dumb, ACCEPT_LINE, "\r", "\n");
```

Comment:
Would adding `"\r\n"` as an additional key binding help with the cr/lf issue on dumb terminals ?

Reply:
That would not be deterministic then. How will the system know if to expect only `\r` or `\r\n`. Remember that there might be no immediate input after the `\r`. Consider the following interaction:

Time `n`: user inputs something that triggers `SELF_INSERT`s, such as `"Hello"`
Time `n+1`: user inputs `\r`. 
Time `n+m`: user inputs a new thing that trigger SELF_INSERTs, not being `\n`.

Surely we cannot wait for a period of unkown `m` to determine whether the `\r` at `n+1` should have triggered `ACCEPT_LINE`.

Reply:
Consider above with this user over a slow connection:

Time `n`: process is piped something that triggers `SELF_INSERT`s, such as `"Hello"`
Time `n+1`: process is piped `\r`
Time `n+m`: process is piped `\n`

Waiting for the `\n` here would be possible (using a short timeout > `m`), but that would mean that a dumb terminal would always wait after a `\r` to see if there is a `\n` following it. Sometimes it could be available, sometimes
not.